logging.go

```diff
@@ -758,6 +761,7 @@ func Log() *zap.Logger {
 }
 
 var (
+	coloringEnabled  = os.Getenv("NO_COLOR") != "1" && os.Getenv("TERM") != "xterm-mono"
```

Comment:
```suggestion
	coloringEnabled  = os.Getenv("NO_COLOR") != "" && os.Getenv("TERM") != "xterm-mono"
```

Reply:
@mholt The example implementation on the no-color.org website linked in the issue shows that if the `NO_COLOR` is set to "0", we should set the coloring to false. I updated the code to follow this implementation. Let me know what you think about it and I can update again accordingly. 
`int
main(int argc, char *argv[])
{
	char *no_color = getenv("NO_COLOR");
	bool color = true;

	if (no_color != NULL && no_color[0] != '\0')
		color = false;

	/* do getopt(3) and/or config-file parsing to possibly turn color back on */
	...
}`

Reply:
Oh, that's actually the null byte, if my C memory isn't underflowing me.

The actual verbiage says:

> Command-line software which adds ANSI color to its output by default should check for a NO_COLOR environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color.

So presumably a value of 0 still counts as "set", but... that is kinda weird to do. I dunno why set the variable at all in that case. :man_shrugging: Maybe we just leave it as-is for now and see if anyone complains (with good reason for setting it to 0)?

Reply:
Oh, I believe you're right, I did not see the text with it. My C knowledge is also very rusty at this point so I missed that. I will update it to conform to the specs.